### ISSUE TITLE
[New Feature]: Integrate Airflow with Cognito authentication (and authorization?)

### ISSUE THREAD

**Issue by LucaCinquini** [unity-sds/unity-sps-prototype#258]
Acceptance Criteria:
o Demonstrated use of Airflow API via Unity API Gateway URL
o Demonstrated login into Airflow UI using the Unity API Gateway URL

Dependencies:
From CS: Unity API Gateway deployed onto the target venues (mcp-venue-dev and mcp-venue-test)

**Comment by GodwinShen** [comment]
@LucaCinquini I see this is still assigned to sprint 24.1.2, but you mentioned in your comment that you will move it to a later sprint.

**Comment by GodwinShen** [comment]
@LucaCinquini are we still waiting on U-CS coordination before we move forward with this ticket?  What do we need to do to get this one moving?

**Comment by LucaCinquini** [comment]
Yes we are. Let's wait until Galen is back and then I hope Ramesh can do this, or it will be a combination or Ramesh and Drew.
